src/Sql/Sql/ManagedInstance/Model/AzureSqlManagedInstanceModel.cs

```diff
@@ -50,7 +50,7 @@ public class AzureSqlManagedInstanceModel
         /// <summary>
         /// Gets or sets the identity of the managed instance.
         /// </summary>
-        public Management.Sql.Models.ResourceIdentity Identity { get; set; }
+        public Management.Sql.Models.ResourceIdentityWithUserAssignedIdentities Identity { get; set; }
```

Comment:
part of your change?

Reply:
@rewongmicrosoft ask Vinit and Shobhit to check it.